features/step_definitions/operators.rb

```diff
@@ -26,6 +26,31 @@
   end
 end
 
+Given /^I wait(?: up to #{NUMBER} minutes)? for all clusteroperators to be ready$/ do |timeout_in_minutes|
+  ensure_admin_tagged
+  timeout_in_minutes ||= 30
+  seconds = Integer(timeout_in_minutes) * 60
+  clusteroperators = BushSlicer::ClusterOperator.list(user: admin)
+  co_hash = {}
+  clusteroperators.each do |co|
+    logger.info("Getting information for #{co.name}...")
+    res = cluster_operator(co.name).wait_till_ready(admin, seconds)
```

Comment:
Looks like in the loop each cluster operator will be waiting 1800 seconds to become ready.

Reply:
the loop will exit on the first failure.